### DISCUSSION TITLE
Manual throttle fixed wing launch mode option

### DISCUSSION THREAD

**Discussion by breadoven** [iNavFlight/inav#8130] [Ideas]
Just wondering if there's any interest in a manual throttle fixed wing launch mode option. Essentially launch mode would just control the attitude during launch, wings level and pitched up to climb, but the throttle would be manually controlled by stick movement not automatically by launch.

So you'd engage Launch mode, raise the stick to whatever power you wanted and throw with INAV just controlling the climb out attitude and you able to control the throttle as you want by moving the stick.

Sound useful, or useless ?

**Comment by Jetrell** [comment]
I'm not against the idea. But I don't know how popular it would be for most users that have only ever known auto launch.

Would it be up to the pilot to disengage the stabilized climb, as with manual launch?
If so, I imagine its biggest advantage would be for those planes that are hard to launch.. but once they get some airspeed, are fine. 
With planes like that, you can't get your thumb on the elevator stick fast enough, after the throw.

**Reply by breadoven** [reply]
It's basically like doing a manual launch but you let INAV stabilise the climb rather than you having to do it (difficult when one hand has just been used to launch, probably the pitch/roll stick hand normally). I only thought of it because I now launch flying wings using a swing launch, shaking to start the motor before throwing rather than relying on the detection to start the motor after I've thrown it which never seems to get it up to speed quickly enough even with all the delays minimised. So why bother with the motor control at all when you can do it manually and just use INAV to control the attitude.

The Launch would end as it does now with the exception of the throttle that would just follow the stick position, i.e. it would just level off and switch to other modes if set.

**Comment by Jetrell** [comment]
Some of the blokes I fly with, tried out this feature today. And they said it was better in their opinion, than the standard auto launch.

But there was one unanimous view, concerning the new 250uS stick abort deadband.. They thought it was too wide for agile models, that have higher rates of rotation on the roll and pitch.
For e.g. when the stick has to be pushed past 250uS + expo... To disable launch mode.. Many of these planes react so quickly, with that much stick applied at the point of abort, that they will almost invert themselves before you're aware of it... Then the model requires manual attitude adjustment from this wild ride.
I was asked if you could please add an adjustable stick abort deadband, to suit the roll / pitch rate of specific models.. Or hard code the deadband no greater than 100uS either side of center stick?

Would you be willing to add this change @breadoven ?

**Reply by breadoven** [reply]
Probably easier just to make it a setting. Could also be based on the raw stick value not the mixed value. I'll have a look at it.
